src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@use './variables' as *;
+
+@mixin oppsite($width) {
+  display: flex;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: $paragraph-width;
```

Comment:
Ensure that the `$paragraph-width` variable is defined in the `variables` file. If it's not defined, the width might not be applied correctly when using the `oppsite` mixin.